tests/integration/template_weekly_comprehensive_v3.cfg

```diff
@@ -110,14 +110,15 @@ ref_final_yr = 1986
 ref_start_yr = 1985
 ref_years = "1985-1986",
 # Include all sets
-# min_case_e3sm_diags_depend_on_climo: "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","annual_cycle_zonal_mean","zonal_mean_2d_stratosphere",
+# min_case_e3sm_diags_depend_on_climo: "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","annual_cycle_zonal_mean","zonal_mean_2d_stratosphere","aerosol_aeronet","aerosol_budget",
 # min_case_e3sm_diags_depend_on_ts: "enso_diags","qbo",
 # min_case_e3sm_diags_diurnal_cycle: "diurnal_cycle",
 # min_case_e3sm_diags_streamflow: "streamflow",
 # min_case_e3sm_diags_tc_analysis: "tc_analysis",
 # min_case_e3sm_diags_tropical_subseasonal: "tropical_subseasonal",
 # TODO: Add "tc_analysis" back in after empty dat is resolved.
-sets = "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","annual_cycle_zonal_mean","zonal_mean_2d_stratosphere","enso_diags","qbo","diurnal_cycle","streamflow","tropical_subseasonal",
+# TODO: Add "aerosol_budget" back in once that's working for v3.
+sets = "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","annual_cycle_zonal_mean","zonal_mean_2d_stratosphere","enso_diags","qbo","diurnal_cycle","streamflow","tropical_subseasonal","aerosol_aeronet",
```

Comment:
I left `"aerosol_budget"` off the weekly v3 test since that won't work at the moment.